templates/admin.html

```diff
@@ -0,0 +1 @@
+{% include 'layout.html' %}
```

Comment:
![image](https://github.com/PatrykDu/popcorn-works-fastapi-fullstack-app/assets/113666877/f3eb32f0-2b21-4f91-be99-1c067b5b2376)

vs

![image](https://github.com/PatrykDu/popcorn-works-fastapi-fullstack-app/assets/113666877/587164b4-dff6-4dcd-a078-7cfaf6ed9719)

Reply:
It suppose to be like that. We don't want to be able to send message as a logged user.